webrtc-stats.html

```diff
@@ -1288,6 +1297,87 @@
                   second and the bitrate is calculated over a 1 second window.
                 </p>
               </dd>
+              <dt>
+                <dfn><code>requestsReceived</code></dfn> of type <span class=
+                "idlMemberType"><a>unsigned long long</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Represents the total number of connectivity check requests received (includes retransmissions).
+                </p>
+              </dd>
+              <dt>
+                <dfn><code>requestsSent</code></dfn> of type <span class=
+                "idlMemberType"><a>unsigned long long</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Represents the total number of connectivity check requests sent (does not include retransmissions).
+                </p>
+              </dd>
+              <dt>
+                <dfn><code>responsesReceived</code></dfn> of type <span class=
+                "idlMemberType"><a>unsigned long long</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Represents the total number of connectivity check responses received.
+                </p>
+              </dd>
+              <dt>
+                <dfn><code>responsesSent</code></dfn> of type <span class=
+                "idlMemberType"><a>unsigned long long</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Represents the total number of connectivity check responses sent.
+                </p>
+              </dd>
+              <dt>
+                <dfn><code>retransmissionsSent</code></dfn> of type <span class=
+                "idlMemberType"><a>unsigned long long</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Represents the total number of connectivity check retransmissions sent.
```

Comment:
Yes. Consent requests are not retransmitted (the next consent request is send with a different transaction id).